app/views/hq/sponsors/index.html.haml

```diff
@@ -0,0 +1,37 @@
+- if @sponsors.empty?
```

Comment:
I still don't see what these guards give us over the clarity that empty table provides.